src/include/stir/ProjDataGEAdvance.h

```diff
@@ -73,7 +73,18 @@ class ProjDataGEAdvance : public ProjData
     Sinogram<float> get_sinogram(const int ax_pos_num, const int sergment_num,const bool make_num_tangential_poss_odd=false) const; 
     Succeeded set_sinogram(const Sinogram<float>& s);
  
-    
+    float get_bin_value(const int segment_num,
```

Comment:
cannot do this. Will generate errors for any decent compiler. That's why i wouldn't force implementation. (there's no way we can test if an ProjDataGEAdvance mods are correct as we don't have such files anymore)